user-guides/ivche1337/Streams-And-Topics-Part2.md

```diff
@@ -0,0 +1,47 @@
+# Streams & topics, Part 2
+
+## Unsubscribe from a stream
+- Unsubscribing from a stream through the Streams page
+- Unsubscribing from a stream through the stream sidebar
+
+* No typos
+* Steps clear and descriptive
+* Screenshots would ease the process
+* I followed the steps easy
+## Change the privacy of a stream
+* No typos
+* Steps clear and descriptive
+* Screenshot would ease the process
+* Begginer friendly
+* I followed the steps easy
+
+## Organize the Streams sidebar
+- Pinning
+- Searching
+- Muting
+
+* I tried all the keyboard shortcuts and they worked
+* The documentation is clear and descriptive
+* Screenshots would ease the process
+* Begginer friendly
+* No typos
+## Pin a stream
+* Easy to use feature
+* Descriptive tutorial
+* Screenshots would ease the process
+* No typos
+## Change the colo of a stream
+* Easy to use feature
+* Begginer friendly
+* Steps are straightforward and clear
+* No typos
+## Message a stream by email   
+* No typos
+* Medium to follow
```

Comment:
Can you explain why this particular section is harder to follow? It would be very helpful for us to improve the documentation.

Reply:
Since there are no screenshots..
As i listed there..

Reply:
Ok that's fine. Maybe this with the screenshot sentence?